hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -84,28 +80,95 @@ Default: 0x00000001 (with 3 SD boot retries to match the current bootloader beha
 The timeout in milliseconds for the entire DHCP sequence before failing the current iteration.  
 Default: 45000  
 Minimum: 5000  
+Version: stable/pieeprom-2020-01-17.bin  
 
 ### DHCP_REQ_TIMEOUT
 The timeout in milliseconds before retrying DHCP DISCOVER or DHCP REQ.  
 Default: 4000  
 Minimum: 500  
+Version: stable/pieeprom-2020-01-17.bin  
 
 ### TFTP_TIMEOUT
 The timeout in milliseconds for an individual file download via TFTP.  
 Default: 15000  
 Minimum: 5000  
+Version: stable/pieeprom-2020-01-17.bin  
 
 ### TFTP_IP
 Optional dotted decimal ip address (e.g. 192.169.1.99) for the TFTP server which overrides the server-ip from the DHCP request.  
 This maybe useful on home networks because tftpd-hpa can be used instead of dnsmasq where broadband router is the DHCP server.
 Default: ""  
+Version: stable/pieeprom-2020-01-17.bin  
+
+### TFTP_PREFIX
+In order to support unique TFTP boot directories for each Pi the bootloader prefixes the filenames with a device specific directory. If neither start4.elf nor start.elf are found in the prefixed directory then the prefix is cleared.
+On earlier models the serial number is used as the prefix, however, on Pi4 the MAC address is no longer generated from the serial number making it difficult to automatically create tftpboot directories on the server by inspecting DHCPDISCOVER packets. To support this the TFTP_PREFIX may be customized to either be the MAC address, a fixed value or the serial number (default).
+
+* 0 - Use the serial number e.g. "9ffefdef/"
+* 1 - Use the string specified by TFTP_PREFIX_STR
+* 2 - Use the MAC address e.g. "DC-A6-32-01-36-C2/"
+Default: 0
+Version: stable/pieeprom-2020-01-17.bin  
+
+### TFTP_PREFIX_STR
+Specify the custom directory prefix string used when TFTP_PREFIX is set to 1. For example:- TFTP_PREFIX_STR=tftp_test/
+Default: ""
+Version: stable/pieeprom-2020-01-17.bin  
+
+## Network Boot
+### Server configuration                                                    
+Network boot requires a TFTP and NFS server to be configured.  See [Network boot server tutorial](bootmodes/net_tutorial.md)
+
+Additional notes:-
+* The MAC address on the Pi4 is programmed at manufacture and is not derived from the serial number.
+```                                                                          
+# mac adddress (ip addr) - it should start with DC:A6:32
+ip addr | grep ether | head -n1 | awk '{print $2}' | tr [a-z] [A-Z]                                                                                                                                                
+# serial number                                                                                                                         
+vcgencmd otp_dump | grep 28: | sed s/.*://g
+```
+
+### Installation - firmware update
+Network boot functionality is included in the 2020-02-13 Raspbian Buster release. However, for advanced boot modes it's normally best to use the latest stable software.
```

Comment:
Just to make it really obvious what the second sentence means, do you think it is maybe worth changing "advanced boot modes" to "network boot"? Also, a minor style issue - not sure what the convention is in the docs, but I would expect contractions to be discouraged, so: it's -> it is. I've also seen the wording "it is recommended" somewhere else, which I think sounds better, so perhaps this:

"However, for network boot is it recommended to use the latest stable software".

Reply:
Advanced boot modes also covers USB mass storage boot

Reply:
Is this paragraph not part of the section that describes how to enable network boot? Apologies if I've read it wrong.